cf/scf-develop.yaml.erb

```diff
@@ -266,6 +275,27 @@ jobs:
         params:
           file: out/version
 
+  - name: package-bits
+    plan:
+      - aggregate:
+        - get: src
+          passed:
+          <% if manual_pull_sources %>
+          - pull-sources
+          <% end %>
+          trigger: true
+        - get: ci
+        - get: s3.certstrap-binary
```

Comment:
If we're putting certstrap inside these things, we should have one for linux and one for darwin. (The two can be done in parallel in the same job, though)

See the certstrap pipeline for one way this could be done.

This may mean changing the existing certstrap s3 resource to name it as linux.